packages/rspack/tests/configCases/errors/loader-syntax-error/index.js

```diff
@@ -1,10 +1,10 @@
 it("should report syntax error", () => {
-  let errored = false;
+	let errored = false;
 	try {
 		require("./lib");
 	} catch (e) {
-    errored = true;
-		expect(e.message).toContain("SyntaxError");
+		errored = true;
+		expect(e.message).toContain("Syntax Error");
```

Comment:
this changed because of swc？

Reply:
@msdlisper can you help explain this change to me?
![image](https://github.com/web-infra-dev/rspack/assets/8898718/c9415e50-3adb-4040-8902-36056a70aeda)

Reply:
This difference is due to the different error messages between babel-jest and @swc/jest. So this change is safe